action.yaml

```diff
@@ -308,10 +308,19 @@ runs:
         branch=$(echo "${GITHUB_REF#refs/heads/}")
         echo "Branch pushed to: $branch"
         git fetch origin $branch
-        files=$(git diff --name-only ${{ github.event.before }} ${{ github.event.after }})
-        echo "Files changed: $files"
+
         DBT_DEPLOY=false
 
+        # case when the triggered event is a new tag creation, then we would need to deploy the dbt project to be on the safe side
```

Comment:
nit: there is also a before null commit on pushing a new branch

Reply:
ohh yeah, good call to include that as well 👍